### PR TITLE
Reintroduce tests for removed SVG attributes.

### PR DESCRIPTION
**Pull request by gsnedders**
These accidentally got nuked in the Blink merge.

### CODE REVIEW

File: tree-construction/tests11.dat [hunk 1]
```diff
@@ -272,6 +272,62 @@
...
+#document
+| <!DOCTYPE html>
+| <html>
+|   <head>
+|   <body>
+|     <svg svg>
+|       contentscripttype=""
+|       contentstyletype=""
+|       externalresourcesrequired=""
+|       filterrest=""
```
Comment (nox) [inline tree-construction/tests11.dat]: `filterres` isn't in the list of SVG attributes to be adjusted any longer. Also, you mistyped it anyway.

File: tree-construction/tests11.dat [hunk 2]
```diff
@@ -272,6 +272,62 @@
...
```
Comment (nox) [inline tree-construction/tests11.dat]: Mistyped.

File: tree-construction/tests11.dat [hunk 3]
```diff
@@ -272,6 +272,62 @@
...
```
Comment (nox) [inline tree-construction/tests11.dat]: Mistyped.

File: tree-construction/tests11.dat [hunk 4]
```diff
@@ -272,6 +272,62 @@
...
+#document
+| <!DOCTYPE html>
+| <html>
+|   <head>
+|   <body>
+|     <math math>
+|       contentscripttype=""
+|       contentstyletype=""
+|       externalresourcesrequired=""
+|       filterrest=""
```
Comment (nox) [inline tree-construction/tests11.dat]: Mistyped.

**Comment by gsnedders** [comment]
Um. Oops. I English, it good.

**Comment by gsnedders** [comment]
OK, hopefully this commit fixes it.

File: tree-construction/tests11.dat [hunk 5]
```diff
@@ -272,6 +272,62 @@
 |       zoomandpan=""
 
 #data
+<!DOCTYPE html><body><svg contentScriptType='' contentStyleType=''externalResourcesRequired='' filterRes=''></svg>
```
Comment (zcorpan) [inline tree-construction/tests11.dat]: Is the lack of space between attributes intentional? (If so, should be in \#errors.)

**Comment by zcorpan** [comment]
LGTM % nit

File: tree-construction/tests11.dat (hunk 5 above)
Comment (gsnedders) [inline tree-construction/tests11.dat]: No.

**Pull request merged**